code/builders/builder-vite/src/vite-config.ts

```diff
@@ -77,6 +77,12 @@ export async function commonConfig(
 
 export async function pluginConfig(options: Options) {
   const frameworkName = await getFrameworkName(options);
+  const { features } = options;
```

Comment:
`features` option is only for transitional features. I propose `test.build` option that is a peer to `features` (at the top level of `main.js` instead.

Reply:
I see this was resolved, what was the resolution?